shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -66,6 +66,8 @@ public class FlutterActivityAndFragmentDelegateTest {
   private FlutterEngine mockFlutterEngine;
   private FlutterActivityAndFragmentDelegate.Host mockHost;
 
+  @SuppressWarnings("deprecation")
+  // Robolectric.setupActivity
```

Comment:
Consider extracting a `Activity GetMockActivity()` or similar that returns `Robolectric.setupActivity(Activity.class)` so you aren't suppressing the entire method.